pkg/packagekit/wix/service.go

```diff
@@ -240,6 +245,5 @@ func cleanServiceName(in string) string {
 		".", "_",
 	)
```

Comment:
forward and back slashes are also invalid

Reply:
Yeah... I should really look up the full rules here, and then make a whitelist. But removing the likely characters was quicker. I added those two.